openupgrade_scripts/scripts/calendar/18.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Models in module 'calendar'---
+---Fields in module 'calendar'---
+calendar     / res.users.settings       / calendar_default_privacy (selection): NEW required, selection_keys: ['confidential', 'private', 'public'], hasdefault: default
+# NOTHING TO DO
+
+---XML records in module 'calendar'---
+DEL ir.actions.act_window: calendar.action_view_start_calendar_sync
+NEW ir.ui.view: calendar.res_users_form_view
+NEW ir.ui.view: calendar.res_users_form_view_calendar_default_privacy
+# NOTHING TO DO
+
+DEL onboarding.onboarding: calendar.onboarding_onboarding_calendar (noupdate)
+DEL onboarding.onboarding.step: calendar.onboarding_onboarding_step_setup_calendar_integration (noupdate)
+# DONE: post-migration: safely delete
```

Comment:
@MiquelRForgeFlow you finally didn't do this in the migration scripts.

Reply:
My bad! Done in https://github.com/OCA/OpenUpgrade/pull/4972.